### DISCUSSION TITLE
2019-06-12: Pytorch 官方翻译奖励-认领中

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#615] [Announcements]
凡事翻译过贡献者，都可以申请得到官方的纪念礼物：
如果有问题的人私聊： QQ 529815144

```
邮件部分内容：
As a start, let us know where we can ship you some PyTorch swag! 
We have stickers, t-shirts, hoodies, and backpacks – let us know what sizes you need.
```
![image](https://user-images.githubusercontent.com/9199175/59322626-99703300-8d09-11e9-9be9-0f07381f42a6.png)

留下联系方式：
QQ: 529815144 (记得先加我QQ)
收货人: 片刻
手机号码: 131xxxx0490
所在地区: 北京市xxx区xxx镇
详细地址: xxx小区xxxx

重点注意： **选择你喜欢礼物和型号**

由于是贡献者，请留下你参与贡献的记录，例如：
（如果没找到，也可以留下  pull resquests  就行 ）
![image](https://user-images.githubusercontent.com/9199175/59322581-62018680-8d09-11e9-88a1-37e35b14dbbf.png)

**Comment by hijkzzz** [comment]
收货人: 胡健
手机号码:  17150301070
所在地区: 湖南省岳阳市岳阳楼区
详细地址: 八字门阳光小区
重点注意： hoodies 或者 t-shirts L码
```
torch.Tensor
torch.sparse
torch.nn.functional
Multiprocessing package - torch.multiprocessing
```

**Comment by infdahai** [comment]
收货人: cluster
手机号码: 15077913432
所在位置: 安徽省合肥市蜀山区
详细地址: 中科大西校区5号楼

new pytorch assignees

物品: 背包(backpack) 型号: 能装电脑

**Comment by kemingzeng** [comment]
#### 地址
- 收货人：Twinkle
- 手机号码：18380116300
- 所在位置：四川省成都市成华区
- 详细地址：电子科技大学（沙河校区）校内18栋

#### 需求
- 背包（backpack）： 大小没概念，正常款，能放下15.6的笔记本最好。

#### 贡献
- 早期大量校验与翻译
- 第一期负责人 :grin:
<img src="https://user-images.githubusercontent.com/20683510/59324365-2cf93200-8d11-11e9-8474-ae71c53d3e51.png" height="150" />

**Comment by friedhelm739** [comment]
收货人: 姚文轩
手机号码: 15566228517
所在位置: 河北省秦皇岛市海港区
详细地址: 燕山大学西校区
贡献：Reinforcement Learning (DQN) Tutorial | @friedhelm739 | 100% ，以及0.3版本NLP部分翻译。
重点注意： 背包，大概能装下13寸电脑；如果没有了帽衫也可以，185的。

**Comment by fanqie03** [comment]
收货人：陈
手机号码：18826538552 
所在位置：广东省深圳市龙岗区坂田街道
详细地址：吉祥花园新村69栋

需求：t-shirt，170的🌹🌹

[Translation with a Sequence to Sequence Network and Attention](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/seq2seq_translation_tutorial.md) | @ mengfu188 | 100％
-- | -- | --

**Comment by jiangzhonglian** [comment]
你们都喜欢啥礼物？？
重点注意： 选择你喜欢礼物和型号

**Comment by PengChen01** [comment]
留个脚印，证明我来过

**Comment by weigq** [comment]
收货人:  wei
手机号码: 13439205165
所在地区: 安徽省合肥市蜀山区稻香村街道
详细地址: 中科大西校区研究生公寓
重点注意： 背包，正常尺寸即可，大概能装下13寸电脑；如果没有了帽衫也可以，170的。
贡献：
早期负责`torch.nn`整块的翻译。
![Annotation 2019-06-12 190849](https://user-images.githubusercontent.com/18133388/59347156-8039a780-8d46-11e9-8522-12c5bab9b56a.jpg)

**Comment by nothingcouldbebetter** [comment]
收货人：张假飞
手机号：15068886433
所在地区：四川省成都市双流区
详细地址：西航港街道学府路24号
重点注意：没有啥审美，背包够用了，按楼上兄弟讲的，帽衫也不错，170-175之间就行了。thx
贡献：
torch的翻译和校验
![87AH$K($~0EV61XH@%EK28N](https://user-images.githubusercontent.com/32367261/59354120-065dea00-8d57-11e9-8242-5a1be9bf2ae1.png)

**Comment by coboe** [comment]
收货人: 谢智
手机号码: 18998475321
所在地区: 广东省深圳市南山区
详细地址: 粤海街道 粤兴三道 华中科技大学深圳产学研基地A座
喜欢礼物和型号： 帽衫或者T恤，165. 没有尺码就要个背包。

0.3 版本参与翻译的，厚着脸皮来要礼物。

![image](https://user-images.githubusercontent.com/6276255/59359440-ec290980-8d60-11e9-943a-359681dc0aab.png)
![image](https://user-images.githubusercontent.com/6276255/59359465-f814cb80-8d60-11e9-8410-1008c6f33db1.png)

**Comment by Cyril-CC** [comment]
收货人: Cyril
手机号码: 17711043934
所在地区: 四川省成都市双流区
详细地址：西航港街道学府路24号
重点注意：没有啥审美+1，背包也行，帽衫也不错(165-170都行）。thx
贡献：
tersor校验
![@_Q@~ FD@L(_8J{2%EHQ3HV](https://user-images.githubusercontent.com/22650332/59400800-3c3cb600-8dcb-11e9-821a-82953923716c.png)

**Comment by PEGASUS1993** [comment]
厚着大脸皮来认领礼物：
收货人：冯宝宝
手机号码：16601110932
所在地址：北京市西城区闹市口大街1号 长安兴融中心南楼
重点注意：没啥特殊审美，能装下15.6笔记本的黑色电脑包就好了。如果没了的话，T-shirt 180/100的 或者185的也成。谢谢啦。

![1560607798(1)](https://user-images.githubusercontent.com/32333727/59552562-1fff6b80-8fbb-11e9-94de-6d897b663b65.jpg)
![1560607814(1)](https://user-images.githubusercontent.com/32333727/59552563-1fff6b80-8fbb-11e9-830f-b2551de012c5.jpg)
![1560607832(1)](https://user-images.githubusercontent.com/32333727/59552564-20980200-8fbb-11e9-9f1b-4999d38ecc1e.jpg)
![1560607859(1)](https://user-images.githubusercontent.com/32333727/59552565-20980200-8fbb-11e9-81ac-e82871a4ccb8.jpg)

**Comment by Smilexuhc** [comment]
收货人：徐昊成
手机号码：17381591565
地址：四川省成都市郫县西源大道2006号电子科技大学清水河校区
T-shirt 2xl或者185，如果有背包就更好啦
第二期负责人＋校验者

**Comment by AllenZYJ** [comment]
收货人:张锴旺
手机号码:13178375076
地址:福建省漳州市平和县东大街124-17号福星茶庄
T-shirt 2xl(180),有背包就更好啦
1-5校验者

**Comment by jiangzhonglian** [comment]
ok   本次奖励收集到此结束，还想要的期待下一次

**Comment by a625687551** [comment]
不是吧  我刚来就结束了。。。。

**Comment by jiangzhonglian** [comment]
> 不是吧 我刚来就结束了。。。。

你先回复。。 看看有没有多的，记得把你的 Pull Requests 留意下，谢谢

**Comment by a625687551** [comment]
收货人: 王先生
手机号码: 16619869942
所在地区: 北京市朝阳区顺北大街与来广营东路交汇处融新科技中心C座18楼
T-shirt 2xl或者185，如果有背包就更好啦
Chatbot Tutorial 翻译者

**Comment by solerji** [comment]
收货人: 刘陆
手机号码: 15222337039
所在地区: 天津市南开区华苑街道天华路天华里教师村2号楼
有背包就最好能装13寸电脑左右大小的背包，没有的话T恤就可以，要S或者说最小码就可以。我就占个位置，啥时候有第二波再来吧。
![image](https://user-images.githubusercontent.com/25458589/61339857-59edb700-a872-11e9-9f06-e3b94eb169d1.png)
solerji  翻译者

**Comment by bdqfork** [comment]
收货人: 刘陆
手机号码: 15222337039
所在地区: 天津市南开区华苑街道天华路天华里教师村2号楼
175的T恤就可以了，谢谢🙏
<img width="688" alt="屏幕快照 2019-07-17 上午10 09 41" src="https://user-images.githubusercontent.com/23702980/61341947-164b7b00-a87b-11e9-89f6-ff0f059be2cf.png">
<img width="704" alt="屏幕快照 2019-07-17 上午10 09 55" src="https://user-images.githubusercontent.com/23702980/61341949-19df0200-a87b-11e9-80fc-8c1c1f755df7.png">
